cli/src/project.rs

```diff
@@ -123,53 +189,101 @@ impl Cache {
     }
 }
 
-/// Project (crate) info
-///
-/// Such as base URL.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Project {
+pub struct Kvdb {
     pub name: String,
-    pub url: String,
 }
 
-impl Project {
-    /// Get project by name, with automatic cache management.
-    ///
-    /// Returns an error if the API request fails or if there are filesystem issues
-    /// with reading/writing the cache.
-    pub async fn one(name: &str) -> eyre::Result<Self> {
-        let cache = Cache::new().await?;
+/// FileConfig is the structure of kinetics.toml
+#[derive(Debug, Clone, Default, Deserialize)]
+struct FileConfig {
```

Comment:
@flyin I'm not sure if we really need the `FileConfig` struct, potentially we can get along with just `Project`. We'll get back to this later.